bundles/org.openhab.core.io.rest.transform/src/main/java/org/openhab/core/io/rest/transform/TransformationConfigurationDTO.java

```diff
@@ -27,14 +29,14 @@ public class TransformationConfigurationDTO {
     public String label;
     public String type;
     public @Nullable String language;
-    public String content;
+    public Map<String, String> configuration;
     public boolean editable = false;
 
-    public TransformationConfigurationDTO(TransformationConfiguration transformationConfiguration) {
-        this.uid = transformationConfiguration.getUID();
-        this.label = transformationConfiguration.getLabel();
-        this.type = transformationConfiguration.getType();
-        this.content = transformationConfiguration.getContent();
-        this.language = transformationConfiguration.getLanguage();
+    public TransformationConfigurationDTO(Transformation transformation) {
```

Comment:
Seems you have missed to also refactor the name of the DTO. 😄

Reply:
Done. The question is if I should also move the resource to `/transformations` instead of `/transformations/configurations`.

Reply:
I would say so, yes. Then it is in line with other resources.